### ISSUE TITLE
Array.prototype.push.apply() error with karma-jasmine unit test

### ISSUE THREAD

**Issue by gokussx4** [ariya/phantomjs#13629]
Using Chrome or IE as the browser type there is no problem. When running the same unit test through phantomJS this error occurs:
```
TypeError: '[object Object]' is not a valid argument for 'Function.prototype.apply' (evaluating 'response.data')
```

Example code:
```
var data = [];
var otherData = ['1','2','3'];
Array.prototype.push.apply(data, otherData);
```

**Comment by zackw** [comment]
Fixed in 2.0.

**Comment by zackw** [comment]
BTW, thank you very much for providing the nice clear and simple test case.
